### PR TITLE
Search ui and results

### PR DESCRIPTION
**Pull request by Jpadilla1**

### CODE REVIEW

File: src/Layout/components/header.js [hunk 1]
```diff
@@ -13,15 +12,21 @@ export default class ThankYou extends Component {
...
-                        <Link to='/'>Home</Link>
+                        <Link to="/">Home</Link>
                     </Navbar.Brand>
                     <Navbar.Toggle />
                 </Navbar.Header>
                 <Navbar.Collapse>
                     <Nav>
-                        <NavItem eventKey={1} href="/donation-form">Donation form link</NavItem>
-                        <NavItem eventKey={2} href="/donation-form/thank-you">Thank You page link</NavItem>
-                        <NavItem eventKey={3} href='/search'>Search page link</NavItem>
```
Comment (nabeards) [inline src/Layout/components/header.js:24]: info: just curious why we seem to hope between <Link> and <NavItem> in this header. :)

**Review by nabeards — APPROVED** [review]
This looks great!

File: src/Layout/components/header.js (hunk 1 above)
Comment (Jpadilla1) [reply src/Layout/components/header.js:24]: So NavItem was causing a full page reload (a regular anchor tag), Link however uses React router and the history API avoiding the full page reload

Comment (nabeards) [reply src/Layout/components/header.js:24]: Ah, good to know! Thanks.

**Pull request merged**